v3.3/glfw/vulkan.c

```diff
@@ -0,0 +1,5 @@
+#include "vulkan.h"
+
+void* getVulkanProcAddr() {
+	return glfwGetInstanceProcAddress;
+}
```

Comment:
How about defining this inline in `vulkan.go`'s `import "C"` block? It's so small it doesn't seem to me to warrant two files of its own.